pkg/controller/servicebindingrequest/service.go

```diff
@@ -170,9 +173,14 @@ func getOwnedResources(
 	error,
 ) {
 	var resources []*unstructured.Unstructured
+BINDING_RESOURCES:
```

Comment:
This is stray text 🤔

Reply:
this is a label used in L182 with continue for better clarity

Reply:
I think `continue` without the label is not confusing.

Reply:
@baijum Looks okay to me. Are you okay with this change?

Reply:
@DhritiShikhar I think the label can be removed.

Reply:
was trying to stay consistent with https://github.com/redhat-developer/service-binding-operator/blob/master/pkg/controller/servicebindingrequest/servicecontext.go#L62